File: Modul-4/Soal-5/PRAK405-2410817210027-HAFIZPERDANA.c
#include <stdio.h>

int main() {
    int n, k, i, j, jumlah, totalJumlah = 0;

    scanf("%d %d", &n, &k);

    for (i=1; i<=n; i++) {
        jumlah = 0;
        for (j=i; j>=1; j--) {
            printf("(%d * %d)", j, k);
            jumlah += j * k;
            if (j > 1) {
                printf(" + ");
            }
        }
        printf(" = %d\n", jumlah);
        totalJumlah += jumlah;
    }

    printf("%d\n", totalJumlah);

    return 0;
}

File: Modul-4/Soal-5/PRAK405-2410817210027-HAFIZPERDANA.py
n, k = map(int, input().split())

totalJumlah = 0

for i in range(1, n+1):
    jumlah = 0
    for j in range(i, 0, -1):
        print(f"({j} * {k})", end="")
        jumlah += j * k
        if j > 1:
            print(" + ", end="")
    print(f" = {jumlah}")
    totalJumlah += jumlah

print(totalJumlah)